src/components/SpectralProfiler/SpectralProfilerComponent.tsx

```diff
@@ -101,7 +150,7 @@ export class SpectralProfilerComponent extends React.Component<WidgetProps> {
             let yCount = 0;
 
             // values are needed to be sorted in incremental order for binary search
-            let values: Array<{ x: number, y: number }> = [];
+            let values: Array<{x: number, y: number}> = [];
```

Comment:
it seems we've previously been inconsistent with the spacing. I'm open to it changing, as long as we're consistent.